src/lib.rs

```diff
@@ -228,6 +231,22 @@ impl<'a> ParserState<'a> {
         // namespaces or class names.
         let symbol = self.read_name(true)?;
 
+        match &symbol.name {
+            &Name::RTTITypeDescriptor(_) |
+            &Name::RTTIClassHierarchyDescriptor(_) |
+            &Name::RTTIBaseClassArray(_) => {
+                if self.input.len() != 0 {
+                    return Err(Error::new("Expected end of input after RTTI name!".to_owned()));
+                }
+
+                return Ok(ParseResult {
+                    symbol: symbol,
+                    symbol_type: Type::None,
+                });
+            },
+            _ => {},
+        };
+
```

Comment:
Instead of consuming the '8' above and having these checks here we should be able to handle '8' below (after the '7' used for Type::CXXVBTable) and maybe add a Type::RTTI or something like that.